components/org.wso2.carbon.identity.auth.service/src/main/java/org/wso2/carbon/identity/auth/service/util/AuthConfigurationUtil.java

```diff
@@ -146,7 +148,59 @@ public void buildClientAuthenticationHandlerControlData() {
         }
     }
 
+    /**
+     * Build intermediate cert validation config.
+     */
+    public void buildIntermediateCertValidationConfigData() {
+
+        OMElement intermediateCertValidationElement = IdentityConfigParser.getInstance().getConfigElement(Constants
+                .INTERMEDIATE_CERT_VALIDATION_ELE);
+        if (intermediateCertValidationElement != null) {
+            isIntermediateCertValidationEnabled = Boolean.parseBoolean(intermediateCertValidationElement
+                    .getAttributeValue(new QName(Constants.CERT_AUTHENTICATION_ENABLE_ATTR)));
+            if (isIntermediateCertValidationEnabled) {
+                //get intermediate cert CNs
```

Comment:
Comments need to start with a capital letter and end with '.' and Space needed between the // and 1st word
                // Get intermediate cert CNs.

Reply:
Fixed with https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/72